Misc/NEWS

```diff
@@ -371,6 +371,8 @@ Library
 - bpo-30068: _io._IOBase.readlines will check if it's closed first when
   hint is present.
 
+- bpo-29447: tempfile's dir argument is now accept path-like objects.
```

Comment:
This isn't accurate since the support was already there implicitly, so just drop the changes to this file or add a comment that tests for path-like objects in tempfile were added to the "Tests" section.